src/map/script.cpp

```diff
@@ -26939,6 +26939,25 @@ BUILDIN_FUNC(macro_detector) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+// ===================================
+// *autoloot(<rate>{, <char_id>});
+// This command set the rate of autoloot.
+// Returns autoloot value on success.
+// ===================================
+BUILDIN_FUNC(autoloot) {
+	map_session_data *sd;
+
+	if (!script_charid2sd(3, sd)) {
+		script_pushint(st, 0);
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	sd->state.autoloot = cap_value(script_getnum(st, 2), 0, 10000);
```

Comment:
Please change this so that wrong usage (< 0 and > 10000) get reported and result in SCRIPT_CMD_FAILURE

Reply:
I added warning if value are invalid, but I think it shouldn't stop the script if wrong value since we could just cap it and continue the script, the capped value doesn't affect the result compare to the value provided by users.